lib/mpl_toolkits/axisartist/angle_helper.py

```diff
@@ -386,12 +386,13 @@ def __call__(self, transform_xy, x1, y1, x2, y2):
 
         # iron out jumps, but algorithm should be improved.
         # Tis is just naive way of doing and my fail for some cases.
-        if self.lon_cycle is not None:
-            lon0 = np.nanmin(lon)
-            lon -= 360. * ((lon - lon0) > 180.)
-        if self.lat_cycle is not None:
-            lat0 = np.nanmin(lat)
-            lat -= 360. * ((lat - lat0) > 180.)
+        with np.errstate(invalid='ignore'):
```

Comment:
Right you are. I guess I was being lazy :smiley: 
I assume that np.unroll() should we np.unwrap?